app/assets/stylesheets/local/alerts.scss

```diff
@@ -0,0 +1,38 @@
+.alert {
```

Comment:
Maybe if we have a bit of a slack in the next sprint, we can speak to some other devs in the department and try to back port this to the `moj_template`.